app/src/main/java/com/github/studydistractor/sdp/createEvent/CreateEventViewModel.kt

```diff
@@ -0,0 +1,327 @@
+package com.github.studydistractor.sdp.createEvent
+
+import android.util.Log
+import androidx.lifecycle.ViewModel
+import com.github.studydistractor.sdp.createActivity.CreateActivityViewModel
+import com.github.studydistractor.sdp.ui.state.CreateEventUiState
+import com.google.android.gms.tasks.Task
+import com.google.android.gms.tasks.Tasks
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.flow.asStateFlow
+import kotlinx.coroutines.flow.update
+import java.text.ParseException
+import java.text.SimpleDateFormat
+import java.util.Calendar
+
+object EventViewModelConstants {
+    const val MAX_NAME_LENGTH = 20
```

Comment:
To what do the max/min values geographically correspond ? Just EPFL or anywhere in the world ?